src/main/java/com/smoothstack/utopiaairlines/controller/AdminController.java

```diff
@@ -0,0 +1,179 @@
+package com.smoothstack.utopiaairlines.controller;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import com.smoothstack.utopiaairlines.entities.Airplane;
+import com.smoothstack.utopiaairlines.entities.Flight;
+import com.smoothstack.utopiaairlines.service.AirplaneService;
+import com.smoothstack.utopiaairlines.service.FlightService;
+
+@RestController
+public class AdminController {
```

Comment:
We should discuss potentially splitting the controllers up based on entity; having them all in one class seems like an organization nightmare.

Reply:
Lets discuss this with everyone once we are ready to move on to next micro service